lms/static/js/spec/groups/views/discussions_spec.js

```diff
@@ -0,0 +1,518 @@
+define(['backbone', 'jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
+    'common/js/spec_helpers/template_helpers',
+    'js/discussions_management/views/discussions',
+    'js/groups/views/course_cohort_settings_notification', 'js/discussions_management/models/course_discussions_detail',
+    'js/discussions_management/views/divided_discussions',
+    'js/discussions_management/views/divided_discussions_course_wide',
+    'js/discussions_management/views/divided_discussions_inline',
+    'js/discussions_management/models/course_discussions_settings'
+],
+    function(Backbone, $, AjaxHelpers, TemplateHelpers, DiscussionsView,
+             CourseCohortSettingsNotificationView, CourseDiscussionTopicDetailsModel,
+             CohortDiscussionsView, CohortCourseWideDiscussionsView, CohortInlineDiscussionsView,
+             CourseDiscussionsSettingsModel) {
+        'use strict';
+
+        describe('Discussions View', function() {
+            var createMockDiscussionsSettingsJson, createDiscussionsView, discussionsView, requests, verifyMessage,
+                createMockDiscussionsSettings, createMockCohortDiscussionsJson, createMockCohortDiscussions,
+                showAndAssertDiscussionTopics;
+
+            // Selectors
+            var inlineDiscussionsFormCss = '.cohort-inline-discussions-form',
```

Comment:
I don't know if we will ever change the UI (beyond RTL and i18n fixes), but it's fine to leave this stuff as-is.